runbotics/runbotics-orchestrator-ui/src/src-app/store/slices/Credentials/Credentials.state.ts

```diff
@@ -2,8 +2,17 @@ import { Credential } from 'runbotics-common';
 
 import { BasicCredentialDto } from '#src-app/views/credentials/Credential/Credential.types';
 
+interface AvailableCredentials {
+    id: string;
+    order: number;
+    credentialId: string;
+    processId: string;
+    credential: Credential;
+}
+
```

Comment:
I think it should be AvailableCredential (singular)

Reply:
Also - if I understand correctly in this object credential is formatted by formatProcessCredentials method (in process-credential.service), so credential: Credential is not true. I think there should be an interface added to the credential.model.ts and should be called ProcessCredential. Happy to discuss, especially if I don't understand it correctly

Reply:
first comment - This is what i get from the database (the format is in the database)

Reply:
second comment - credential: Credential makes sense because:
![image](https://github.com/user-attachments/assets/866eb87c-2860-4073-8f29-ef187f83b91c)
![image](https://github.com/user-attachments/assets/7b3e41dc-fd4a-4fee-9183-8dd4de3b18f3)
Although, you're right, ProcessCredential looks like a perfect type for that, I'll change it thanks!

Reply:
done